micro-ui/web/micro-ui-internals/packages/modules/core/src/components/PrivacyComponent.js

```
@@ -35,12 +35,12 @@ const PrivacyComponent = ({ onSelect, formData, control, formState, ...props })
 
   return (
     <React.Fragment>
-      <div className="digit-privacy-checkbox">
+      <div className="digit-privacy-checkbox digit-privacy-checkbox-align">
         <CheckBox label={t("ES_BY_CLICKING")} checked={isChecked} onChange={handleCheckboxChange} id={"privacy-component-check"}></CheckBox>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)